src/core/ext/filters/client_channel/subchannel.cc

```diff
@@ -893,12 +892,10 @@ void Subchannel::SetConnectivityStateLocked(grpc_connectivity_state state,
   health_watcher_map_.NotifyLocked(state, status);
 }
 
+// Caller must call WeakUnref after OnRetryTimer returns
```

Comment:
I don't think we need this comment, since this isn't where anyone will be looking to consider ref-counting.

Reply:
Removed